doc/cephadm/monitoring.rst

```diff
@@ -70,3 +70,10 @@ cluster.
 * To enable the dashboard's prometheus-based alerting, see :ref:`dashboard-alerting`.
 
 * To enable dashboard integration with Grafana, see :ref:`dashboard-grafana`.
+
+Enabling the RBD-Image monitoring
+---------------------------------
+
+Currently due to performance reasons the monitoring of rbd images is per default disabled. For more information please
+see :ref:`prometheus-rbd-io-statistics`. If the preparations for this were not made then the overview and details
```

Comment:
maybe: If disabled, then the overview and details dashboards will stay empty in Grafana...